truffle/src/main/java/org/jruby/truffle/language/constants/LookupConstantNode.java

```diff
@@ -74,9 +74,17 @@ protected RubyConstant lookupConstant(
         if (!isVisible) {
             throw new RaiseException(coreExceptions().nameErrorPrivateConstant(module, name, this));
         }
+        if(constant != null && constant.isDeprecated()){
```

Comment:
Please try to fit in with the existing style as a general rule :smiley: 
Maybe we should do this more automatically, like with mx eclipseformat, what do you think?